python_digest/__init__.py

```diff
@@ -172,10 +172,10 @@ def _check_required_parts(parts, required_parts):
     return len(missing_parts) == 0
 
 def _build_object_from_parts(parts, names):
-    obj = type("", (), {})()
+    obj = type(b"", (), {})()
```

Comment:
this works on py3 as well? For some reason in another pr I thought htis needed to be a text type for both py2 and py3

Reply:
Whoops - it needs to be binary type in py2 and text type in py3.  Fixed now.